custom/icds_reports/templates/icds_reports/icds_app/program-summary.directive.html

```diff
@@ -6,7 +6,7 @@
                 <span class="breadcrump-element" ng-bind="$ctrl.label"></span>
             </div>
             <div class="row">
-                <span class="location-breadcump" ng-click="$ctrl.getDisableIndex() < 0 && $ctrl.moveToLocation('national', -1)">National </span>
+                <span class="location-breadcump" ng-click="$ctrl.haveAccessToAllLocations && $ctrl.moveToLocation('national', -1)">National </span>
```

Comment:
`ng-click` is the function that gets fired when you click on the breadcrumb. is the idea here that if the first part returns `false` js will short-circuit and not evaluate the second part?

Reply:
yes If we does not have access to it